src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceBase.cs

```diff
@@ -87,17 +79,6 @@ public async Task<T> ExecuteScalarQueryAsync<T>(string query, CancellationToken
         /// <inheritdoc/>
         public abstract void UpdateDatabase(string databaseName);
 
-        /// <inheritdoc/>
-        public abstract CompletionItem[] GetAutoCompleteSuggestions(ScriptDocumentInfo queryText, Position index, bool throwOnError = false);
-        /// <inheritdoc/>
-        public abstract Hover GetHoverHelp(ScriptDocumentInfo scriptDocumentInfo, Position textPosition, bool throwOnError = false);
-        
-        /// <inheritdoc/>
-        public abstract DefinitionResult GetDefinition(string queryText, int index, int startLine, int startColumn, bool throwOnError = false);
-
-        /// <inheritdoc/>
-        public abstract ScriptFileMarker[] GetSemanticMarkers(ScriptParseInfo parseInfo, ScriptFile scriptFile, string queryText);
-
```

Comment:
These 4 were moved to KustoIntellisenseHelper since they're related to that and didn't _need_ to be on KustoDataSource